### ISSUE TITLE
Laravel method assertSeeText() failing because of JavaScript comment

### ISSUE THREAD

**Issue by fterrani** [silviolleite/laravel-pwa#6]
The Laravel method `TestResponse::assertSeeText()` fails when turning on **laravel-pwa**.

It is because this method uses `strip_tags()` to remove HTML tags before checking if a given text is found in the page.

The bug is simply caused by a this JavaScript comment in **views/modules/laravelpwa/meta.blade.php**:

`// <--- THIS BIT IS REQUIRED`

Strangely, `<--` apparently confuses `strip_tags()`. Like this:

``` php
$str = "aaa bbb <-- ccc ddd";
echo strip_tags( $str ); // Displays "aaa bbb " (tested in a PHP 7.2.4 sandbox)
```

**Comment by silviolleite** [comment]
Hello @fterrani I have fixed it on new release 1.0.4. https://github.com/silviolleite/laravel-pwa/releases/tag/1.0.4. Can you update and try it again.

**Comment by fterrani** [comment]
Hello @silviolleite ,

Looks like it's working for me. Thanks for the fix :)
